src/commcare_cloud/ansible/roles/thp/tasks/disk_scheduler.yml

```diff
@@ -0,0 +1,24 @@
+- stat:
```

Comment:
This file looks really similar to src/commcare_cloud/ansible/roles/kernel_tune/tasks/disk_scheduler.yml. Are they both supposed to be in this PR?

Reply:
That was one stale file hanging around. it's removed now.